lib/Core/rgbToHex.js

```diff
@@ -0,0 +1,8 @@
+function componentToHex(c) {
+  var hex = c.toString(16);
+  return hex.length === 1 ? "0" + hex : hex;
+}
+
+export default function rgbToHex(r, g, b) {
```

Comment:
Cesium's `Color` class can do this: https://cesiumjs.org/Cesium/Build/Documentation/Color.html#toCssColorString

Reply:
Cool - I did look at that class and was suprised it couldn't do what I wanted - I read that method as converting a Cesium Color to a css string.